### DISCUSSION TITLE
Help with validating new model structure/set-up after addition of new organ compartment?

### DISCUSSION THREAD

**Discussion by eburg007** [Open-Systems-Pharmacology/Forum#903] [Questions & Problems]
Hello all,

Using MoBi, I modified the original PBPK model structure to include a new "Lymph Node" organ. This compartment represents all lymph node structures within the body, and collects lymphatic fluid through new neighborhoods connecting each organ's interstitial to the lymph node interstitial. The lymph is then returned to the blood stream by connection of the lymph node interstitial to the venous blood plasma.

If possible, would someone be able to take a look at the model structure and verify that I've set everything up correctly? 
[TFV_LN_NHP zip.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/6523425/TFV_LN_NHP.zip.zip)

Happy to answer any other questions/provide details.
(I utilized the forum discussions github.com/Open-Systems-Pharmacology/MoBi/issues/1 and github.com/Open-Systems-Pharmacology/Forum/issues/302 to set up the new compartment).

**Comment by StephanSchaller** [comment]
Hi @eburg, I think this looks almost right.
I believe you might have an (accidental) error in the total lymph flow calculation. While the sum formula is correct, it will also add the "Lymph flow rate" from the Lymph not which is again defined as an explicit sum of all organ lymph flows:
![image](https://user-images.githubusercontent.com/25477643/121959250-d2ae6400-cd64-11eb-89d9-7c0f7700862a.png)

So the transporter referencing the sum formula of "total lymph flow" in the neighborhood effectively calculates 2x total lymph flow for the lymph flow "lymph node to plasma".
You do not have to set the lymph flow of the lymph node organ to the explicit sum of all lymph flows as this parameter is not used for the transporter. Just set the lymph node lymph flow to a constant value (if you have a value from literature) reflecting the actual lymph fluid flow from lymph node plasma to lymph node intestitial (as in all other organs).
Then your model should be correct.

Hope this helps,
Stephan
